extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -575,6 +575,20 @@ angular.module('PaperUI.controllers.configuration', [ 'PaperUI.constants' ]).con
     $scope.hasProperties = function(properties) {
         return util.hasProperties(properties);
     }
+    
+    $scope.showDescription=function(channel,channelType){
+        var popup = $mdDialog.alert({
+            title: channel.label?channel.label:channel.channelType?channel.channelType.label:channel.id,
+            textContent: channel.description?channel.description:channel.channelType?channel.channelType.description:'No description available',
```

Comment:
Might be. But then again, the description is localized and thus should be the "no description" message, if possible (which it isn't ;-)).
At many other places, a long press does nothing, so I think it is an expected situation that "nothing happens" means "nothing available" and not "all broken" ;-)